website/blog/2022-09-28-analyst-to-ae.md

```diff
@@ -0,0 +1,178 @@
+|            | **Group A**        | **Group B**        | **Group C**        |
+| 2022-08-01 | 34                 | 60                 | 61                 |
+| 2022-08-02 | 77                 | 86                 | 37                 |
+| 2022-08-03 | 71                 | 9                  | 6                  |
+| 2022-08-04 | 63                 | 87                 | 10                 |
+| 2022-08-05 | 99                 | 9                  | 86                 |
+| 2022-08-06 | 41                 | 55                 | 19                 |
+| 2022-08-07 | 100                | 79                 | 66                 |
+| 2022-08-08 | 79                 | 95                 | 1                  |
+
+### Model format needed for the BI tool
+
+| Date       | Group   | Daily Active Users |
+|------------|---------|--------------------|
+| 2022-08-01 | Group A | 34                 |
+| 2022-08-01 | Group B | 60                 |
+| 2022-08-01 | Group C | 61                 |
+| 2022-08-02 | Group A | 77                 |
+| 2022-08-02 | Group B | 86                 |
+| 2022-08-02 | Group C | 37                 |
+| 2022-08-03 | Group A | 71                 |
+| 2022-08-03 | Group B | 9                  |
+| 2022-08-03 | Group C | 6                  |
+
+## Loop 2: Reacting to unexpected results
+
+Analysts are the first (and sometimes only) line of defense to catch data quality issues. Once a dataset is aggregated down to a single number to answer a business question, it’s often impossible to know if there’s an incorrect filter or wrong set of logic.
+
+When a model is built for the first time, all parties often “don’t know what they don’t know”. As the Analyst digs into the curated dataset, they will find updates that need to be made to the model. Hence, our second point:
+
+<h3 align="center">Analysts question everything.</h3>
+
+`Group A` for `Active Users` may be focused on `Participant` roles. The stakeholder had explained to the Analyst that they want to exclude any `Admins` that will have a different experience in the product.
+
+| User_Id | Location   | Role        | Level | Zone |
+|---------|------------|-------------|-------|------|
+| 123     | California | Editor      | AAA   | 1    |
+| 427     | Utah       | Participant | ABA   | 1    |
+| 864     | Georgia    | Admin       | CCC   | 3    |
+| 975     | New York   | Participant | CAB   | 4    |
+| 211     | Florida    | Participant | CBB   | 3    |
+| 936     | Montana    | Admin       | ABA   | 2    |
+| 787     | Nevada     | Participant | ACC   | 2    |
+
+A Data Engineer working off of a “build list” will add a filter for `WHERE Role = 'Participant'`. During the data validation step, the Analyst would discover that there is actually a third `Role` of `Editor` that no-one was aware of. This would create a loop where the Data Engineer would have to edit the model to replace it with `WHERE Role != 'Admin'`.
```

Comment:
```suggestion
A Data Engineer working off of a “build list” will add a filter for `WHERE Role = 'Participant'`. During the data validation step, the Analyst would discover that there is actually a third `Role` of `Editor` that no one was aware of. This would create a loop where the Data Engineer would have to edit the model to replace it with `WHERE Role != 'Admin'`.
```